src/selection.ts

```diff
@@ -0,0 +1,62 @@
+export enum SelectionTypes {
```

Comment:
I'm also not sure I understand the issue with reducing the "number of imports." In my above snippet, would I not need to still import `SelectionType` in `b.ts` to be able to designate it as the type of `a`? Thanks for your patience as I learn TypeScript, I appreciate it!